### PR TITLE
Add a test for associating a project with a mentor object.

### PR DESCRIPTION
**Pull request by Sami-2000**
One super basic test of the addProject function for the mentor object. (hopefully this function in the future may be covered by a more complicated test) Follow the file organization and format from the starter project.

### CODE REVIEW

File: src/test/java/com/google/opensesame/MentorTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  private static final String MENTOR_1_GTHUB = "Mentor_1_GitHub_ID";
+  private static final String PROJECT_1 = "Project 1";
+
+  @Test
+  public void addProjects() {
+    MentorObject mentor1 = new MentorBuilder()
+        .name(MENTOR_1_NAME)
+        .gitHubID(MENTOR_1_GITHUB)
+        .buildMentor();
+    mentor1.addProject(PROJECT_1);
```
Comment (Obinnabii) [inline src/test/java/com/google/opensesame/MentorTest.java:21]: Very nice, I like the mentor builder

File: src/test/java/com/google/opensesame/MentorTest.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import java.io.IOException;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class PersonTests {
+  private static final String MENTOR_1_NAME = "Mentor 1";
+  private static final String MENTOR_1_GTHUB = "Mentor_1_GitHub_ID";
```
Comment (billyeh) [inline src/test/java/com/google/opensesame/MentorTest.java:12]:
```suggestion
  private static final String MENTOR_1_GITHUB = "Mentor_1_GitHub_ID";
```

File: src/test/java/com/google/opensesame/MentorTest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  private static final String PROJECT_1 = "Project 1";
+
+  @Test
+  public void addProjects() {
+    MentorObject mentor1 = new MentorBuilder()
```
Comment (billyeh) [inline src/test/java/com/google/opensesame/MentorTest.java:17]: As there's not currently a `mentor2` I suggest naming this variable just `mentor`.

File: src/test/java/com/google/opensesame/MentorTest.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class PersonTests {
+  private static final String MENTOR_1_NAME = "Mentor 1";
+  private static final String MENTOR_1_GTHUB = "Mentor_1_GitHub_ID";
+  private static final String PROJECT_1 = "Project 1";
```
Comment (billyeh) [inline src/test/java/com/google/opensesame/MentorTest.java:13]: For these variable names I suggest something like `MENTOR_NAME`, `MENTOR_GITHUB_ID`, and `PROJECT_ID` or such. Reasoning: there's no `MENTOR_2_NAME` right now so there's no need to specify `1`. If we did add a second mentor, we could just make an array and then they'd be indexed in the array and we still wouldn't need `1`, `2`, etc.

And it'd be good to add `_ID` to the ID variables so it's more obvious what they represent.

Comment (Sami-2000) [reply src/test/java/com/google/opensesame/MentorTest.java:13]: Thank you!

**Comment by Richie78321** [comment]
@Sami-2000 It seems like this test leads to a compilation error which I am now receiving when working from the master branch. Please remember to test your code before you submit it.

**Pull request merged**